wgpu-rs/src/lib.rs

```diff
@@ -776,8 +776,8 @@ impl CommandEncoder {
 }
 
 impl<'a> RenderPass<'a> {
-    pub fn set_bind_group(&mut self, index: u32, bind_group: &BindGroup) {
-        wgn::wgpu_render_pass_set_bind_group(self.id, index, bind_group.id);
+    pub fn set_bind_group(&mut self, index: u32, bind_group: &BindGroup, offsets: &[u32]) {
```

Comment:
The API spec states an *optional sequence of u64*, however `gfx-hal` expects `u32`, so that's why I used that here. I also opted for a slice instead of an `Option`, as that seems more idiomatic, and the underlying API also expects a slice.

Reply:
slice is good here, no need for `Option`